requirements/torch.txt

```diff
@@ -1,4 +1,5 @@
 torch==1.6.0
 torchvision==0.7.0
 torchtext==0.7.0
-torchaudio==0.6.0
\ No newline at end of file
+torchaudio==0.6.0
+captum==0.2.0
```

Comment:
@abishekchiffon Is this update needed in other requirements files as well? torch_cpu.txt, torch_cu101.txt

Reply:
Moved the captum requirements to common.txt